notifikasjon/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/notifikasjon/river/SakFerdigLoeserTest.kt

```diff
@@ -55,7 +55,8 @@ class SakFerdigLoeserTest : FunSpec({
             mockAgNotifikasjonKlient.nyStatusSak(
                 id = expected.sakId,
                 status = SaksStatus.FERDIG,
-                statusTekst = "Mottatt - Se kvittering eller korriger inntektsmelding"
+                statusTekst = "Mottatt - Se kvittering eller korriger inntektsmelding",
+                nyLenkeTilSak = "$linkUrl/im-dialog/kvittering/${actual.forespoerselId}"
```

Comment:
Jeg lurer på om vi bør bruke expected her?

```suggestion
                nyLenkeTilSak = "$linkUrl/im-dialog/kvittering/${expected.forespoerselId}"
```